printers/virtual.go

```diff
@@ -3,20 +3,21 @@ package printers
 import "github.com/ingridhq/zebrash/elements"
 
 type VirtualPrinter struct {
-	StoredGraphics          map[string]elements.StoredGraphics
-	LabelHomePosition       elements.LabelPosition
-	NextElementPosition     elements.LabelPosition
-	DefaultFont             elements.FontInfo
-	DefaultOrientation      elements.FieldOrientation
-	DefaultAlignment        elements.TextAlignment
-	NextElementFieldBlock   *elements.FieldBlock
-	NextElementFieldData    interface{}
-	NextFont                *elements.FontInfo
-	NextDownloadFormatName  string
-	NextHexEscapeChar       byte
-	NextElementFieldReverse bool
-	LabelReverse            bool
-	// TODO: Add barcode field default handling
+	StoredGraphics            map[string]elements.StoredGraphics
+	LabelHomePosition         elements.LabelPosition
+	NextElementPosition       elements.LabelPosition
+	DefaultFont               elements.FontInfo
+	DefaultOrientation        elements.FieldOrientation
+	DefaultAlignment          elements.TextAlignment
+	NextElementFieldBlock     *elements.FieldBlock
+	NextElementFieldData      interface{}
+	NextFont                  *elements.FontInfo
+	NextDownloadFormatName    string
+	NextHexEscapeChar         byte
+	NextElementFieldReverse   bool
+	LabelReverse              bool
+	DefaultBarcodeModuleWidth int
+	DefaultBarcodeHeight      int
```

Comment:
what do you think about grouping these values into some struct and putting it in elements package?

Reply:
makes sense, I will do that